### PR TITLE
Update: Improved React handling of logic and improved a11y

### PR DESCRIPTION
**Pull request by cahirodoherty-learningpool**
[//]: # (Link the PR to the original issue)
Fixes [ADAPT-10572](https://learningpool.atlassian.net/browse/ADAPT-10572)

### Fix
* Removed duplicated function calls

### Update
* Imported singletons
* Moved View logic onto template
* Simplified the html structure

### CODE REVIEW

File: templates/openTextInput.jsx [hunk 1]
```diff
@@ -17,21 +20,25 @@ export default function OpenTextInput(props) {
...
             placeholder={placeholder}
             maxLength={_allowedCharacters}>{userAnswer}
           </textarea>
         </div>
-        <div className='opentextinput__item-modelanswer opentextinput__item-textbox'>
+        <div
+          className={classes([
+            'opentextinput__item-modelanswer',
+            'opentextinput__item-textbox',
+            !_shouldShowModelAnswer && 'hidden'
```
Comment (eleanor-heath) [inline templates/openTextInput.jsx:39]: can hidden be renamed to is-hidden please

File: less/opentextinput.less [hunk 2]
```diff
@@ -16,13 +13,8 @@
     width: 100%;
   }
 
-  &__hide-modelanswer {
+  .hidden {
```
Comment (eleanor-heath) [inline less/opentextinput.less:16]: .is-hidden here then too

**Review by eleanor-heath — APPROVED** [review]

**Review by ethan-lp — APPROVED** [review]

File: templates/openTextInput.jsx (hunk 1 above)
Comment (eleanor-heath) [reply templates/openTextInput.jsx:39]: or you could potentially use u-display-none from the framework

Comment (cahirodoherty-learningpool) [reply templates/openTextInput.jsx]: Thanks! Implemented now

**Pull request merged**